AppAuth.xcodeproj/project.pbxproj

```diff
@@ -2036,7 +2036,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
What's the reason to bump deployment target?

Reply:
No reason, the wizard did it. The Xcode 10 ui only has down to 8.0 in the dropdown but you can type-in whatever version you want. I reverted it to 7.0.